### DISCUSSION TITLE
How to batch insert products?

### DISCUSSION THREAD

**Discussion by ghost** [saleor/saleor#8566] [Q&A]
I would like to batch insert multiple products into Saleor. I would be very grateful for an mutation example.

**Comment by karolkielecki** [comment]
@ivjuris Hi! There is no official solution. One of our community members created a small python wrapper around Saleor API. It allows you to easily create your product. Check it out here: https://github.com/grll/saleor-gql-loader. 

But since the release of that wrapper, we've released Saleor 3.0 beta. Not sure if there are any issues between that wrapper and 3.0.
